openy_af4_vue_app/dist/activity_finder_4.css

```diff
@@ -1 +0,0 @@
fieldset-component .fieldset-title .title{text-transform:uppercase;font-size:15px;line-height:1.5rem}.sort-radios-component .option{display:-webkit-box;display:-ms-flexbox;display:flex;height:50px}.sort-radios-component .option input[type=radio]{display:none}.sort-radios-component .option input[type=radio]+label{cursor:pointer;display:-webkit-box;display:-ms-flexbox;display:flex;position:relative;margin:auto 0;padding-left:50px;color:#231f20;font-weight:400;font-size:12px;line-height:18px}.sort-radios-component .option input[type=radio]+label:before{content:"";display:block;position:absolute;left:0;height:20px;width:20px;margin:auto 15px;border:2px solid #636466;border-radius:50%;-ms-flex-item-align:center;align-self:center}.sort-radios-component .option input[type=radio]:checked+label:after{content:"";display:block;position:absolute;top:4px;left:20px;width:10px;height:10px;background-color:#0060af;border-radius:50%}.sort-select-component{position:relative;display:-webkit-box;display:-ms-flexbox;display:flex;-webkit-box-align:center;-ms-flex-align:center;align-items:center}.sort-select-component select{height:48px;border:1px solid #636466;border-radius:8px;color:#231f20;min-width:260px;width:auto;padding:10px 5px 10px 10px;-webkit-appearance:none;-moz-appearance:none;appearance:none;z-index:1;background:transparent;outline:none;font-size:18px;line-height:1.5rem}.sort-select-component select::-ms-expand{display:none}.sort-select-component select option{padding:10px;font-size:18px;line-height:1.5rem}.sort-select-component .fa,.sort-select-component .svg-inline--fa{position:absolute;right:0;font-size:12px;margin:0 14px}.search-form-component{border:1px solid #636466;color:#636466;display:-webkit-box;display:-ms-flexbox;display:flex;margin:0 auto;border-radius:8px;padding:10px 5px 10px 10px}.search-form-component .form-control{border:none;font-size:18px;line-height:18px;height:38px;padding:0}.search-form-component .btn{color:#fff;width:40px;padding:6px 12px;border:none;font-size:18px;line-height:1;background-color:transparent;border-radius:8px}.search-form-component .btn span{color:#231f20}.search-form-component .btn .fa{font-size:18px}.search-form-component ::-webkit-input-placeholder{font-size:18px;font-style:normal}.search-form-component :-moz-placeholder,.search-form-component ::-moz-placeholder{font-size:18px;font-style:normal}.search-form-component :-ms-input-placeholder{font-size:18px;font-style:normal}.no-results-component .actions{display:-webkit-box;display:-ms-flexbox;display:flex;-webkit-box-orient:vertical;-webkit-box-direction:normal;-ms-flex-direction:column;flex-direction:column;margin:20px 0}.no-results-component .actions .btn{font-weight:bolder;line-height:50px;padding:0 10px;border-radius:5px;background-color:#92278f;color:#fff;margin-bottom:20px}.no-results-component .actions .btn:last-child{margin-bottom:0}.landing-content{padding-top:0!important}.activity-finder__wrapper{position:relative}.row:after,.row:before{height:0;width:0}
```

Comment:
This css file was deleted by mistake?

Reply:
@rollki yeah, thank you for your attention.Fixed